digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadAddressChangesDAO.scala

```diff
@@ -189,4 +194,11 @@ object RoadAddressChangesDAO {
     }
   }
 
+  def insertNewRoadChange(newRoad: ProjectLink): Unit = {
```

Comment:
personally i would have added newroad seq to delta case class, stored changes to there and used above insertDeltaToRoadChangeTable so we could get all the changes inserted in same DAO method (and if i am not mistaken all rows are inserted at the same time on insertDeltaToRoadChangeTable and in sqlu they are added one by one.